topics/variant-analysis/tutorials/trio-analysis/workflows/main_workflow.ga

```diff
@@ -3,7 +3,8 @@
     "annotation": "Trio Analysis using Synthetic Datasets from RD-Connect GPAP",
     "creator": [
         {
-            "class": "Organization",
+            "class": "Person",
+            "identifier": "",
```

Comment:
you can fill your Orcid here if you like.

Reply:
Thanks!